proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+## Your Proposal Title
+
+* **Owners:**
+  * Daniel Schürmann
+
+* **Implementation Status:** `Not implemented`
+
+* **Related Issues and PRs:**
+
+* **Other docs or links:**
+  * https://ev.kde.org/corporate/statutes/
+  * https://phabricator.kde.org/T12954
+
+* This design doc shall be used to file our Articles of Assoziation 
+
+## Draft
+
+# Articles of Association - Mixxx e.V.
+
+As a non-profit corporation incorporated under German law, Mixxx e.V. is required to declare its intentions and by-laws in a formal "Articles of Association" document. The original and legally binding version is the German version. The following translation is for information only.
+
+## Clarification of Terminology
+
+1. Computer programs considered as free software or "open source software" are put into the public domain free of charge by the originator, allowing others the freedom to use, study, adapt, copy, and improve the program for the common good.
+2. Developing free software involves research, compilation of principles and concepts, and their testing through programming and testing of free software.
+3. "Mixxx" is a DJ software developed as free software and available to the public free of charge.
+
+## 1. Company Name, Registered Office, Financial Year
+
+1. The Association is registered as Mixxx e.V.
+2. The registered offices are located in Berlin. If no fixed office is established, it will be managed from the residence of the Board member acting as general manager.
+3. It is listed under the appropriate register in the District Court’s Register of Associations.
+4. The financial year is the calendar year.
+
+## 2. Purpose of the Association
+
+1. The Association’s purpose is explain the basics in digital DJ-ing to beginners and educate software developemen students and other interested in digital sound processing and online colaboration. For this pupose we promote and distribute free and open source DJ software, particularly "Mixxx". This gives the equality of opportunity in starting with dgital DJ-ing in bedrooms, school parties and up to professional applications. In addition it allows free exchange of knowledge, for education, science and research.
+2. The articles aim to:
+   1. Distribute Mixxx as a free and open source software
+   2. Distribute Manuals and Guides
+   3. Organize user support and education 
+   4. Mentor students during interneships for learning software developmen.
```

Comment:
```suggestion
   4. Mentor students during interneships for learning software development.
```

Reply:
I suggest we add a pre-commit hook with codespell to this repo :)